src/fheroes2/army/army_ui_helper.cpp

```diff
@@ -88,12 +120,26 @@ void fheroes2::drawMiniMonsters( const Troops & troops, int32_t cx, const int32_
             fheroes2::Blit( monster, output, cx + offset, cy + offsetY + monster.y() );
             text.draw( cx + chunk - text.width() - offset, cy + 23, output );
         }
-        else {
+        else if ( slotsToSkip == 0 ) {
             const int offsetY = 28 - monster.height();
```

Comment:
To be consistent with `fheroes2::Blit()` input variables types lets set the ёште32_её type for this variable:
```suggestion
            const int32_t offsetY = 28 - monster.height();
```